src/elements/Input/Input.js

```diff
@@ -237,8 +237,8 @@ class Input extends Component {
         {actionPosition === 'left' && actionElement}
         {labelPosition !== 'right' && labelElement}
         {createHTMLInput(input || type, { defaultProps: htmlInputProps })}
-        {actionPosition !== 'left' && actionElement}
         {Icon.create(this.computeIcon())}
+        {actionPosition !== 'left' && actionElement}
```

Comment:
I would prefer a test that the render order is as it should be.  However, a comment will do in this case.  We use this convention when encountering gotchas like this:

```
// Heads up!
// A short comment about something very important.
```

Let's notify our future selves that SUI CSS requires the icon to be between these two elements.